sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationExec.scala

```diff
@@ -275,7 +252,7 @@ object HiveScriptIOSchema {
 }
 
 /**
- * The wrapper class of Hive input and output schema properties
+ * The wrapper class of Hive script transformation input and output schema properties
  */
 case class HiveScriptIOSchema (
```

Comment:
I'm still not sure we need to have this derived class for io schema (https://github.com/apache/spark/pull/29085#discussion_r454784809); We cannot define a container case class for this io schema then moving the methods of `HiveScriptIOSchema` into private ones in `HiveScriptTransformationExec`? For example;

```
// Shared container case class for io schema between `Hive/SparkScriptTransformationExec`
case class ScriptIOSchema (
    inputRowFormat: Seq[(String, String)],
    outputRowFormat: Seq[(String, String)],
    inputSerdeClass: Option[String],
    outputSerdeClass: Option[String],
    inputSerdeProps: Seq[(String, String)],
    outputSerdeProps: Seq[(String, String)],
    recordReaderClass: Option[String],
    recordWriterClass: Option[String],
    schemaLess: Boolean)

case class HiveScriptTransformationExec(
    input: ...)
  extends BaseScriptTransformationExec with HiveInspectors {
...
private def initInputSerDe(input: Seq[Expression]): Option[(AbstractSerDe, StructObjectInspector)] = { ...}


```

Reply:
Nice suggestion, this way can avoid a lot duplicated code.